torch/jit/_monkeytype_config.py

```diff
@@ -0,0 +1,127 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Optional, Iterable, List, Dict
+from collections import defaultdict
+
+_IS_MONKEYTYPE_INSTALLED = True
+try:
+    import monkeytype
+    from monkeytype import trace as monkeytype_trace
+    from monkeytype.db.base import CallTraceThunk, CallTraceStore, CallTraceStoreLogger
+    from monkeytype.config import default_code_filter
+    from monkeytype.tracing import CallTrace, CodeFilter
+except ImportError:
+    _IS_MONKEYTYPE_INSTALLED = False
+    print("Warning: monkeytype is not installed. Please install https://github.com/Instagram/MonkeyType"
```

Comment:
Yes we need to move that. Thanks for pointing. How about we move it here: https://github.com/pytorch/pytorch/pull/55376/files#diff-c05f6af41a2d9c7ec7a2b15088259fb74763f7d1406da70f324fc6b20af47427R886

This will make sure the warning is printed only when we try to use PDT

```
if monkeytype_trace:
        monkeytype_config = JitTypeTraceConfig(type_trace_db)
        with monkeytype_trace(monkeytype_config):
            for example_input in example_inputs:  # type: ignore[union-attr]
                obj(*example_input)
else:
    print("Warning: monkeytype is not installed. Please install https://github.com/Instagram/MonkeyType "
          "to enable Profile-Directed Typing in TorchScript. Refer "
          "https://github.com/Instagram/MonkeyType/blob/master/README.rst to install MonkeyType. ")
```

Reply:
If you use `warnings.warn` then we can adjust how often it is printed.

Reply:
Now fixed.